src/components/Menu/config/footerConfig.ts

```diff
@@ -0,0 +1,76 @@
+import { FooterLinkType } from '@pancakeswap/uikit'
+import { ContextApi } from 'contexts/Localization/types'
+
+export const footerLinks: (t: ContextApi['t']) => FooterLinkType[] = (t) => [
+  {
+    label: t('About'),
+    items: [
+      {
+        label: t('Contact'),
+        href: 'https://docs.pancakeswap.finance/contact-us',
+      },
+      {
+        label: t('Blog'),
+        href: 'https://pancakeswap.medium.com/',
+      },
+      {
+        label: t('Community'),
+        href: 'https://docs.pancakeswap.finance/contact-us/telegram',
+      },
+      {
+        label: t('CAKE token'),
+        href: 'https://docs.pancakeswap.finance/tokenomics/cake',
+      },
+      {
+        label: '—',
```

Comment:
Not that you're asking for my input but: This label has a on hover property which underlines it.

Reply:
Good catch, will fix this on next UIKit release.